docs/compute-resources.md

```diff
@@ -217,6 +320,113 @@ spec:
 Here, the minimum of the "max" values is the output "max" value, and likewise for "default" and "defaultRequest".
 The maximum of the "min" values is the output "min" value.
 
+### Configure Task-level Compute Resources on TaskRun with LimitRange
```

Comment:
after thinking about this a bit more, I think it would be better to just leave out this section for now. It works basically the same way as what's already described in the docs.

Reply:
yap. np for this.